django/views/templates/default_urlconf.html

```diff
@@ -1,44 +1,84 @@
-<!DOCTYPE html>
-<html lang="en"><head>
-  <meta http-equiv="content-type" content="text/html; charset=utf-8">
-  <meta name="robots" content="NONE,NOARCHIVE"><title>{{ title }}</title>
-  <style type="text/css">
-    html * { padding:0; margin:0; }
-    body * { padding:10px 20px; }
-    body * * { padding:0; }
-    body { font:small sans-serif; }
-    body>div { border-bottom:1px solid #ddd; }
-    h1 { font-weight:normal; }
-    h2 { margin-bottom:.8em; }
-    h2 span { font-size:80%; color:#666; font-weight:normal; }
-    h3 { margin:1em 0 .5em 0; }
-    h4 { margin:0 0 .5em 0; font-weight: normal; }
-    table { border:1px solid #ccc; border-collapse: collapse; width:100%; background:white; }
-    tbody td, tbody th { vertical-align:top; padding:2px 3px; }
-    thead th {
-      padding:1px 6px 1px 3px; background:#fefefe; text-align:left;
-      font-weight:normal; font-size:11px; border:1px solid #ddd;
-    }
-    tbody th { width:12em; text-align:right; color:#666; padding-right:.5em; }
-    #summary { background: #e0ebff; }
-    #summary h2 { font-weight: normal; color: #666; }
-    #explanation { background:#eee; }
-    #instructions { background:#f6f6f6; }
-    #summary table { border:none; background:transparent; }
-  </style>
-</head>
-
-<body>
-<div id="summary">
-  <h1>{{ heading }}</h1>
-  <h2>{{ subheading }}</h2>
-</div>
-
-<div id="instructions">
-  <p>{{ instructions|safe }}</p>
-</div>
-
-<div id="explanation">
-  <p>{{ explanation|safe }}</p>
-</div>
-</body></html>
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
```

Comment:
I don't have much expertise, but reading https://stackoverflow.com/a/6771584, I think we can drop this since we don't need to support outdated versions of IE.